docs/how-to/installation/aws.txt

```diff
@@ -286,6 +294,84 @@ Listing Clusters
       -  AWS profile used for deploying cluster resources.
       -  default
 
+Printing the default ``master.yaml`` template
+---------------------------------------------
+
+.. code::
+
+   det deploy aws dump-master-yaml-template
+
+.. _aws-master-yaml-template:
+
+******************************
+ Custom master.yaml templates
```

Comment:
This should lie under  ``det deploy`` tool because the document only works with ``det deploy aws``